Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -366,7 +369,7 @@ extension Stripe.Subscription {
   public static let canceling = mock
     |> \.cancelAtPeriodEnd .~ true
 
-  public static let canceled = mock
+  public static let canceled = canceling
```

Comment:
This immediately made the account page render test for canceling subscriptions fail 😄